### PR TITLE
make changes per chads email; add retries; log errors

### PR DESCRIPTION
**Pull request by atvaccaro**
# Description

Closes https://github.com/cal-itp/data-infra/issues/1776
Closes https://github.com/cal-itp/data-infra/issues/1780

* Adds retries as the resource level
* Tried to PATCH resource file types but got a 404, likely a permissions thing
* Add field description authorities
* Only pass through the numeric value for EPSG
* Uppercase CSV headers

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation
- [ ] agencies.yml

## How has this been tested?
I have an email thread going with CDOT about the 500s we're experiencing; the retries help but do not fix the problem.

### CODE REVIEW

File: warehouse/models/gtfs_schedule_latest_only/_gtfs_schedule_latest_only.yml [hunk 1]
```diff
@@ -8,16 +8,20 @@ models:
       Each row is a cleaned row from an agency.txt file.
       Definitions for the original GTFS fields are available at:
       https://gtfs.org/reference/static#agencytxt.
+    meta:
+      ckan.authority: https://developers.google.com/transit/gtfs/reference#agencytxt
```
Comment (lauriemerrell) [inline warehouse/models/gtfs_schedule_latest_only/_gtfs_schedule_latest_only.yml:12]: These should reference gtfs.org not the Google Developers site

File: warehouse/models/gtfs_schedule_latest_only/_gtfs_schedule_latest_only.yml [hunk 2]
```diff
@@ -8,16 +8,20 @@ models:
...
     columns:
       - &calitp_itp_id
         name: calitp_itp_id
         description: '{{ doc("column_calitp_itp_id") }}'
         meta:
+          ckan.authority: "Source: GTFS Files for each transit agency in CA, captured nightly by the Cal-ITP team. URL and ITP ID source is https://github.com/cal-itp/data-infra/blob/main/airflow/data/agencies.yml"
```
Comment (lauriemerrell) [inline warehouse/models/gtfs_schedule_latest_only/_gtfs_schedule_latest_only.yml:18]: This is going to become obsolete very soon, but ok for now I guess

File: warehouse/models/gtfs_schedule_latest_only/_gtfs_schedule_latest_only.yml [hunk 3]
```diff
@@ -68,6 +72,8 @@ models:
       Each row is a cleaned row from an attributions.txt file.
       Definitions for the original GTFS fields are available at:
       https://gtfs.org/reference/static#attributionstxt.
+    meta:
+      ckan.authority: https://developers.google.com/transit/gtfs/reference#attributionstxt
```
Comment (lauriemerrell) [inline warehouse/models/gtfs_schedule_latest_only/_gtfs_schedule_latest_only.yml:76]: Same comment as above these should all be gtfs.org

File: warehouse/scripts/publish.py [hunk 4]
```diff
@@ -264,78 +286,81 @@ def _generate_exposure_documentation(
...
+            contact_organization="Caltrans",
+            contact_position="Cal-ITP",
+            contact_name="Hunter Owens",
+            contact_email="hunter.owens@dot.ca.gov",
+            public_access_level="Public",
+            access_constraints=None,
+            use_constraints="Creative Commons 4.0 Attribution",
+            data_life_span=None,
+            caltrans_link=None,
+            data_standard="https://developers.google.com/transit/gtfs",
```
Comment (lauriemerrell) [inline warehouse/scripts/publish.py:323]: gtfs.org

**Review by lauriemerrell — CHANGES_REQUESTED** [review]
few text issues

File: warehouse/models/gtfs_schedule_latest_only/_gtfs_schedule_latest_only.yml (hunk 1 above)
Comment (atvaccaro) [reply warehouse/models/gtfs_schedule_latest_only/_gtfs_schedule_latest_only.yml:12]: Ah yeah weird, tbh I did not realize they were different in our current metadata.

**Review by lauriemerrell — APPROVED** [review]
want to change the spec ref too

**Review by lauriemerrell — CHANGES_REQUESTED** [review]
ugh sorry meant to request changes again

**Review by lauriemerrell — APPROVED** [review]

**Comment by atvaccaro** [comment]
Waiting to merge until we get a response from OpenGov and potentially test against CKAN without 500s.

**Comment by atvaccaro** [comment]
The lint is fine locally, I'm unsure what happened. Going to merge and open a new PR to address linting in general.

**Pull request merged**